src/docs/PML/PMLUserGuide.adoc

```diff
@@ -2898,35 +2880,68 @@ NOTE: Your CSV file should be in a table format. See the example below.
+It also enables:
 
-*Task Variables:*
+- Automatic identification of the best-suited method for encoding each categorical column, when no encoding method is selected (Auto mode).
+- Data type recognition: identification of the data type of each column (categorical or numerical).
+- Creation of summary statistics for each column: missing values, minimum, maximum, average, zeros, and cardinality.
+- Editing of the data structure: modification of column information (name, type, category, etc.), deletion of a column, etc.
+
+This workflow can be used:
+
+- Stand-alone such that the results can be saved in the User Data Space or locally.
+- In a larger workflow where the results will be sent to the next connected task.
 
-.Import_Data_Interactive_Task variables
+NOTE: For further information, please check the subsection <<AutoFeat>>.
+
+
+.Import_Data_And_Automate_Feature_Engineering variables
 [cols="2,5,2"]
 |===
 | *Variable name* | *Description* | *Type*
-| `TASK_ENABLED`
-| If False, the task will be ignored, it will not be executed.
-| Boolean (default=True)
+3+^|*Workflow variables*
 | `IMPORT_FROM`
-| Selects the type of data source.
+| Selects the method/protocol to import the data source.
 | List [PA:URL,PA:URI,PA:USER_FILE,PA:GLOBAL_FILE] (default=PA:URL)
 | `FILE_PATH`
-| Inserts a file path/name.
+| Inserts the path/name of the file that contains the dataset.
 | String
 | `FILE_DELIMITER`
 | Defines a delimiter to use.
 | String (default=;)
 | `LIMIT_OUTPUT_VIEW`
-|  Specifies how many rows of the dataframe will be previewed in the browser to check each task results.
+|  Specifies how many rows of the dataframe will be previewed in the browser to check the encoding results.
 | Int (-1 means preview all the rows)
+| `NATIVE_SCHEDULER`
+| Name of the Native Scheduler node source to use when the workflow tasks must be scheduled on a native scheduler such as SLURM, LSF, etc.
+| String (default=empty)
+| `NATIVE_SCHEDULER_PARAMS`
+|Parameters given to the native scheduler (SLURM, LSF, etc) while requesting a ProActive node used to execute the workflow tasks.
+| String (default=empty)
+| `NODE_ACCESS_TOKEN`
+|If not empty, the workflow tasks will be run only on nodes that contains the specified token.
+| String (default=empty)
+| `NODE_SOURCE_NAME`
+|If not empty, the workflow tasks will be run only on nodes belonging to the specified node source.
+| String (default=empty)
+| `WORK_DIR`
+| Defines the working directory for the data space used to transfer files automatically between the workflow tasks.
+| String
+| `CONTAINER_PLATFORM`
+| Specifies the type of container platform to be used (no container, docker, singularity, or podman).
+| String (default=docker)
+| `CONTAINER_GPU_ENABLED`
+| If True, containers will run based on images containing libraries that are compatible with GPU.
+| Boolean (default=False)
+| `CONTAINER_IMAGE`
+| Specifies the name of the image that will be used to run the workflow tasks.
+| String (default=Empty).
```

Comment:
Please remove the AI workflows common variables. They are already described in this section https://trydev2.activeeon.com/doc/PML/PMLUserGuide.html#_ai_workflows_common_variables 
NATIVE_SCHEDULER
NATIVE_SCHEDULER_PARAMS
...
CONTAINER_IMAGE